### ISSUE TITLE
Sytax Changed?

### ISSUE THREAD

**Issue by mrmasterwill** [Archetapp/In-App-Purchases-Swift-3-in-Xcode-#1]
When I enter SwiftyStoreKit.verifyReceipt(....) it also needs a 'using:' argument, what does this mean? If I just remove it I get an error.

**Comment by partialdata** [comment]
Yeah it seems swiftystorekit has changed their api a bit can you update this project to reflect that?
